packages/blocks/src/list-block/list-model.ts

```diff
@@ -28,6 +36,29 @@ export const ListBlockSchema = defineBlockSchema({
       'affine:edgeless-text',
     ],
   },
+  toModel: () => new ListBlockModel(),
 });
 
-export type ListBlockModel = SchemaToModel<typeof ListBlockSchema>;
+export class ListBlockModel extends BlockModel<ListProps> {
+  override flavour!: 'affine:list';
+
+  override text!: Text;
+
+  constructor() {
+    super();
+
+    this.propsUpdated.on(({ key }) => {
```

Comment:
instead of use `this.propsUpdated`, use

```ts
const dispose = effect(() => {
  const type = this.model.$type.value;
  handleType(type);
})
this.deleted.once(dispose);
```